.github/workflows/kvrocks.yaml

```diff
@@ -254,6 +254,16 @@ jobs:
             ./build/kvrocks
             ./coredumps/*
 
+  docker:
+    name: Check Docker image
+    needs: [check-and-lint]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/build-push-action@v3
```

Comment:
Verify docker build per PR, but for amd64 only. We don't have too much different when compile on amd64 or arm64.